docs/dev-guide/iframe.md

```diff
@@ -71,8 +125,9 @@ const options = {
 const api = new JitsiMeetExternalAPI(domain, options);
 ```
 
-You can override options set in [config.js] and [interface_config.js] via **configOverwrite** and **interfaceConfigOverwrite** respectively.
-For example:
+You can override options set in the [config.js] file and the [interface_config.js] file using the **`configOverwrite`** and **`interfaceConfigOverwrite`** objects, respectively.
+
+For example, to enable the **`filmstripOnly`** interface mode use the following:
```

Comment:
Remove this last sentence, since the example no longer does that.

Reply:
Removed sentence and referenced example

Reply:
The example was OK, just the reference to filmstip only.

Reply:
Added example back into documentation